### ISSUE TITLE
Typo in constant name

### ISSUE THREAD

**Issue by manyk** [Cyberworld-builders/settlement-calculator#1]
It should be SETTLEMENT_CALCULATOR_PATH and not SETTLEMENT_CALCULATOR__PATH.

https://github.com/Cyberworld-builders/settlement-calculator/blob/62948c446ead4294eab8aed4e07452c5cdb3d59b/src/Settlement_Calculator.php#L14

**Comment by jaylong255** [comment]
good catch. if you want to open up a pull request, i'll review and merge it. if not i'll fix it next time i'm in there
